driver/labjack/writer.h

```diff
@@ -135,7 +135,8 @@ struct WriterConfig {
                 return;
             }
             channel.data_type = channel_info.data_type;
-            channels.emplace_back(channel);
+
+            if(channel.enabled) channels.emplace_back(channel);
```

Comment:
Fix formatting. Also, why are you still retrieving the channel and updating it in the initial state map if it is disabled? Why not just check if the channel is disabled on 130 and return early?

Reply:
resolved